homeassistant/components/squeezebox/browse_media.py

```diff
@@ -262,21 +263,30 @@ def _get_item_thumbnail(
 
     if track_id:
         if internal_request:
-            item_thumbnail = player.generate_image_url_from_track_id(track_id)
-        elif item_type is not None:
-            item_thumbnail = entity.get_browse_image_url(
-                item_type, item["id"], track_id
-            )
+            return cast(str, player.generate_image_url_from_track_id(track_id))
```

Comment:
Why do we need to cast? The method is typed as `def generate_image_url_from_track_id(self, track_id: int) -> str:`

Reply:
Is the library maybe missing a [`py.typed` marker](https://typing.python.org/en/latest/spec/distributing.html#packaging-type-information)?